Dockerfile.standard_job

```diff
@@ -16,4 +16,5 @@ RUN /reconfigure_apt_sources.sh
 #   sudo: used commonly by tests
 #   virtualenv: required in order to prepare the rpc-gating virtualenv
 RUN apt-get update && apt-get install -y bzip2 curl git-core libxml2-utils openssh-client python-minimal python-pip python-virtualenv python-yaml sudo virtualenv
+RUN apt-get update && apt-get install -y bzip2 curl git-core libxml2-utils openssh-client python-minimal python-pip python-virtualenv python-yaml python3-minimal python3-pip python3-virtualenv python3-yaml sudo virtualenv
```

Comment:
This duplicates the line above. Please edit the line above, and ensure that the packages are in alphabetical order. Also, update the comment above to describe why each package is required for future reference.